README.md

```diff
@@ -145,10 +144,18 @@ systemctl status multipathd
 multipath -ll
 ```
 
-#### 3. Configure storage system connectivity
-3.1. Define the hostname of each Kubernetes node on the relevant storage systems with the valid WWPN(for Fibre Channel) or IQN(for iSCSI) of the node.
+#### 3. Enable support for volume Snapshots on your Kubernetes Cluster.
+##### 3.1 Install the Snapshot CRDs, once per cluster.
+Use the 'kubectl create -f config/crd' command to create each of the yaml files located in https://github.com/kubernetes-csi/external-snapshotter/tree/master/config/crd.
+Apply each of the yamls, using the 'kubectl apply - f' command.
+##### 3.2 Install the Common Snapshot Controller, once per cluster.
+Use the 'kubectl create -f deploy/kubernetes/snapshot-controller' command to create each of the yaml files located in https://github.com/kubernetes-csi/external-snapshotter/tree/master/deploy/kubernetes/snapshot-controller.
```

Comment:
If we want to support different namespace add "Update the namespace to an appropriate value for your environment"